src/Ticket.php

```diff
@@ -1259,12 +1259,28 @@ public function prepareInputForUpdate($input)
                     switch ($a) {
                         case 'user':
                              $additionalfield           = '_additional_' . $t . 's';
-                             $input[$additionalfield][] = ['users_id' => $input['_' . $a . 's_id_' . $t]];
+                             $k = '_' . $a . 's_id_' . $t;
+                             if (!isset($input[$additionalfield])) {
+                                $input[$additionalfield] = [];
+                             }
+                             if (!isset($input[$additionalfield]['users_id'])) {
+                                $input[$additionalfield]['users_id'] = [];
+                             }
+                             if (is_array($input[$k])) {
+                                 $input[$additionalfield]['users_id'] = array_merge($input[$additionalfield]['users_id'] ?? [], $input[$k] ?? []);
+                             } else {
+                                 $input[$additionalfield][] = ['users_id' => $input['_' . $a . 's_id_' . $t]];
+                             }
```

Comment:
With this, `$input[$additionalfield]['users_id']` may still be both an array or a string. Why not always use an array ?